optaweb-vehicle-routing-frontend/pom.xml

```diff
@@ -26,25 +26,46 @@
   </parent>
 
   <artifactId>optaweb-vehicle-routing-frontend</artifactId>
-  <packaging>pom</packaging>
+  <packaging>war</packaging>
 
   <name>OptaWeb Vehicle Routing Frontend</name>
 
   <properties>
     <sonar.sources>src</sonar.sources>
-    <sonar.test.inclusions>optaweb-vehicle-routing-frontend/**/*test.ts,optaweb-vehicle-routing-frontend/**/*test.tsx</sonar.test.inclusions>
+    <sonar.test.inclusions>
+      optaweb-vehicle-routing-frontend/**/*test.ts,optaweb-vehicle-routing-frontend/**/*test.tsx
+    </sonar.test.inclusions>
     <sonar.typescript.lcov.reportPaths>coverage/lcov.info</sonar.typescript.lcov.reportPaths>
+
+    <!-- Frontend properties -->
+    <frontend-maven-plugin.version>1.7.5</frontend-maven-plugin.version>
```

Comment:
Please change the property name to `version.frontend.plugin` to be consistent with kie-parent and jboss-parent naming convention for version properties.